base_mail_sender_bcc/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+This module insert an email address which is set as reply_to into BCC column.
+When reply_to was empty, sender's email address is added to the bcc column and reply_to.
+
+Original module was created by Akretion and adjusted by Quartile limited.
+https://apps.odoo.com/apps/modules/10.0/base_mail_sender_bcc/
+
+This module has migrated into custom-modules to develop functions and mark up License info.
```

Comment:
```suggestion
This module does the following when an email is being built:

- Sets the same value as 'Reply To' in 'Bcc' if 'Reply To' is available.
- Otherwise, populates 'Bcc' based on the system parameter setting (key: email_bcc).

The module was originally created by Akretion https://apps.odoo.com/apps/modules/10.0/base_mail_sender_bcc/
and adjusted by Quartile limited.
```